### ISSUE TITLE
Consistently hitting validateDbConstraints error while importing a user

### ISSUE THREAD

**Issue by ashokgelal** [FusionAuth/fusionauth-issues#709]
I'm trying to import just one user for testing. I'm using basically the same data provided [here](https://fusionauth.io/docs/v1/tech/apis/users#import-users) but I keep getting `This is most likely due to a unique key constraint which would indicate one or more of the users in the import request already exist in FusionAuth. Re-attempt the request with additional validation by using the [validateDbConstraints] property. If you have already enabled the additional validation and you still receive this error, please open a bug report.`

I'm getting this error both using a desktop REST Client (I'm using Paw of Mac) as well as using .net core client. I've made sure that there are no duplicates (in fact, I only have 1 user in the database, the admin user).

I tried another POST endpoint, creating an application, and that seems to be working just fine.

I've a feeling that there is something else wrong but FA API thinks that it is a DB constraint error.

![CleanShot 2020-06-29 at 09 33 55](https://user-images.githubusercontent.com/401055/86025733-c7471900-b9eb-11ea-853f-4820a22d5dc2.png)

**Comment by robotdan** [comment]
Can you run the Import request with the `validateDbConstraints` enabled and see what the output is?

**Comment by ashokgelal** [comment]
Sorry I missed out an important detail:

With the following settings, I am able to import it on a local instance but when I try to do the same thing over to a test server served over https, I run into this import issue. I've successfully created an application on this server though.

![CleanShot 2020-06-29 at 09 42 00](https://user-images.githubusercontent.com/401055/86026847-2fe2c580-b9ed-11ea-8c56-50c57154066e.png)

**Comment by ashokgelal** [comment]
I've set `validateDbConstraints` to false in .net client but I just tried setting `validateDbConstraints` to true in Paw and I got the same error.

**Comment by robotdan** [comment]
Can you show me the import request JSON when you enable `validateDbConstraints` ?

**Comment by ashokgelal** [comment]
Here it is: https://cln.sh/PZz9to

**Comment by robotdan** [comment]
That screenshot looks like it is sending text, can you confirm you are sending the body as `JSON` with the `Content-Type` set to `application/json`?

**Comment by ashokgelal** [comment]
I believe it is (I just switched to Text to take the screenshot to make it easy to see the data I'm sending over): https://cln.sh/L4taxq

**Comment by robotdan** [comment]
A couple of options. 

1. Ensure an application exists with the Id specified in the registration by `applicationId`
2. Try the import w/out unicode in the registration username. If you are on MySQL, it is possible your db is not configured properly for unicode.

**Comment by ashokgelal** [comment]
1. Application exists with the ID specified (I've made sure of it)
2. It's s Postgres database and I've confirmed that it doesn't contain any unicode characters.

**Comment by mooreds** [comment]
Do the server logs on the test server show any interesting error messages?

**Comment by ashokgelal** [comment]
Unfortunately, nothing interesting. FusionAuth is running in a docker container and I was hoping that FA would spit out some log on each access but doesn't seem to log anything for an API request. Is there a way to enable the log?

**Comment by mooreds** [comment]
Sorry, I'm a bit confused. Have you tried loading this data with 'Mausbar' rather than 'Mausbår' for the username and seen the same issue?

Can you attach the log files that show up when you run this?

I don't think there's a great way to enable more detailed debugging log output.

**Comment by ashokgelal** [comment]
So after digging into it more, I found out that the culprit was DigitalOcean's hosted Postgres database combined with the fact that the database schema needs to be imported first to make it work with FA (different issue). Since this was only for testing/QA, I'm now running both the database and FA on a single server and I was able to import all the users. I'm still not sure why it doesn't work with a hosted database though. Thanks for your help!
